broadinstitute/pearl
# AR-95 -- adding anchor link capability for OurHealth FAQ

This adds a navbar link for the FAQ, which links to the FAQ section in the main page.  To do so, this adds an anchor-tag-linking library.  The library only adds about 1K to the bundle size.   This also continues, but by no means finishes, the process of making the typescript types on our navbar links a bit more robust

TO TEST:
1. restart Admin and Particiapnt API apps
2. `./scripts populate_portal.sh ourhealth`
3. go to landing page
4. confirm FAQ appears in navbar
5. confirm clicking on it takes you to the FAQ from various places in the app
